src/main/jbake/content/hawkular-clients/index.adoc

```diff
@@ -1,48 +1,96 @@
 = Clients
 Thomas Heute
 2016-07-07
+:icons: font
 :description: Hawkular clients
 :jbake-type: page
 :jbake-status: published
 
-== Client applications/plugins
-
-=== Grafana plugin
-
-link:https://github.com/hawkular/hawkular-grafana-datasource[GitHub repository]
-
-Only supports Metrics.
-
-=== Wildfly Agent
-
-link:https://github.com/hawkular/hawkular-agent[GitHub repository]
-
-Works with Hawkular Metrics and Hawkular Services
-
-=== ManageIQ
-
-link:http://www.manageiq.org[ManageIQ]
-
-=== Heapster
-
-link:https://github.com/kubernetes/heapster[GitHub repository]
-
-== Client Libraries
-
-=== Go
-
-link:https://github.com/hawkular/hawkular-client-go[GitHub repository]
-
-Only supports Metrics at this point.
-
-=== Python
-
-link:https://github.com/hawkular/hawkular-client-python[GitHub repository]
-
-Only supports Metrics at this point.
-
-=== Ruby
-
-link:https://github.com/hawkular/hawkular-client-ruby[GitHub repository]
-
-Only supports Metrics at this point.
+= Client applications/plugins
+
+[cols="6*", options="header"]
+|===
+|Project
+|Description
+|Documentation
+|Project Link
+|Download
+|Compatibility
+
+|*ManageIQ*
+|Management Application, also used to manage Middleware Servers
+|link:manageiq/docs/quickstart-guide/[Quickstart guide]
+|link:http://manageiq.org[ManageIQ Website]
+|link:http://manageiq.org/download/[Downloads]
+|Hawkular Services > 0.0.4
+
+
+|*Grafana plugin*
+|Plugin to display metrics in Grafana
+|link:https://github.com/hawkular/hawkular-grafana-datasource/blob/master/README.adoc[README]
+|link:https://github.com/hawkular/hawkular-grafana-datasource[GitHub repository]
+|
+| Hawkular Services
+Hawkular Metrics
+
+|*WildFly agent*
+|Agent to embed within a WildFly server which feeds Hawkular Services with metrics, inventory and allows to execute operations
+|
+|link:https://github.com/hawkular/hawkular-agent[GitHub repository]
+|
+| Hawkular Services
+Hawkular Metrics
+
+|*Heapster*
+|Heapster sink to store container clusters metrics inside Hawkular
+|
+|link:https://github.com/kubernetes/heapster[GitHub repository]
+|
+|Hawkular Metrics
+
+|===
+
+= Client libraries
+
+
+[cols="6*", options="header"]
+|===
+|Language
+|Documentation
+|Project Link
+|Download
+|Compatibility
+|Comment
+
+|*Go*
+|link:https://github.com/hawkular/hawkular-client-go/blob/master/README.adoc[README]
+|link:https://github.com/hawkular/hawkular-client-go[GitHub repository]
+|
+|Metrics API
+|
+
+|*Ruby*
+|link:https://github.com/hawkular/hawkular-client-ruby/blob/master/README.rdoc[README]
+|link:https://github.com/hawkular/hawkular-client-ruby[GitHub repository]
+|
+|Metrics API,
+Inventory API
```

Comment:
Also Alerts/Events and Operations.
I pretend this is the most advanced library and under very active development by the Hawkular team itself.